_topic_maps/_topic_map.yml

```diff
@@ -1499,9 +1499,7 @@ Topics:
   Topics:
   - Name: About the Kubernetes NMState Operator
     File: k8s-nmstate-about-the-k8s-nmstate-operator
-  - Name: Observing node network state
-    File: k8s-nmstate-observing-node-network-state
```

Comment:
I don't fully understand how all this works, but it seems like we're kind of orphaning this file since it's no longer referenced anywhere. It seems like this file should either be included or the content moved into the updating page if that's going to be the combined place for this (which seems fine to me).

Reply:
I have included the content from this in k8s-nmstate-updating-node-network-config.adoc as there seemed to be duplication

Reply:
New section is: 
- Name: About the Kubernetes NMState Operator
    File: k8s-nmstate-about-the-k8s-nmstate-operator
  - Name: Observing and updating node network configuration
    File: k8s-nmstate-updating-node-network-config
  - Name: Troubleshooting node network configuration
    File: k8s-nmstate-troubleshooting-node-network